hyper_run.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/zsh
```

Comment:
You did not mention this file in README so I assume you can remove it.

Reply:
Yes

Reply:
I think this file is still there.

Reply:
removed